### ISSUE TITLE
Add bodies to Spine object

### ISSUE THREAD

**Issue by AleBles** [orange-games/phaser-spine#4]
So we can do physics on them, as requests in #3.

**Comment by keysim** [comment]
Awesome !

**Comment by AleBles** [comment]
Let's use Spine's SkeletonBounds for this

**Comment by iangregsondev** [comment]
Hi, any update on this ? Interested in have a physics body attached although I think this will need to come as part of the spine-ts update of the plugin ?

**Comment by AleBles** [comment]
@appsolutegeek You are correct, for me it doesn't make much sense to add features like these anymore on a runtime that's not 'officialy' supported anymore, might as well put the extra effort into getting it done with spine-ts. The only problem there is that I don't see it happening quite soon. I'll have to see whatever I can manage in my off hours :(

**Comment by iangregsondev** [comment]
@AleBles Ok, if I can be of help, let me know.  I was checking here https://github.com/EsotericSoftware/spine-runtimes/tree/master/spine-ts/core/src  as I believe we have to use the spine-ts core. I must admit though without a little help I would find it difficult to start. Although the implementation of the Canvas here https://github.com/EsotericSoftware/spine-runtimes/tree/master/spine-ts/canvas/src seems fairly straight forward but it would be nice to be able to support canvas and webgl.  Also has far as the canvas version it seems to draw directly to a canvas object (context)

**Comment by AleBles** [comment]
@appsolutegeek All help is welcome, I started whipping something up in the spine-ts branch, and you are correct that the spine-ts canvas directly draws on the context. I  the current setup if Overwrite PIXI's _renderCanvas method to do spine's rendering. It shows onscreen, but now is the harder part of translating everything to pixi/phaser :x
